packages/styles/scss/components/quote/_quote.scss

```diff
@@ -44,6 +44,9 @@
     &__mark {
       @include carbon--type-style('quotation-01', true);
       @include carbon--font-family('serif');
+      @include carbon--breakpoint-down('md') {
+        left: 1.4rem;
```

Comment:
Wondered if we can put a comment here explain where the number came from.

Reply:
actually, this value was chosen by visually stay the closest as possible from the quotation text. Do you think it's important to add a comment explaining this?

Reply:
I think it is worth adding a note for other developers that view this file in the future since it doesn't follow our normal conventions of the spacing scale.